topdown/http.go

```diff
@@ -52,9 +52,12 @@ var allowedKeyNames = [...]string{
 	"timeout",
 	"cache",
 }
-var allowedKeys = ast.NewSet()
 
-var requiredKeys = ast.NewSet(ast.StringTerm("method"), ast.StringTerm("url"))
+var (
+	allowedKeys              = ast.NewSet()
+	requiredKeys             = ast.NewSet(ast.StringTerm("method"), ast.StringTerm("url"))
+	httpSendLatencyMetricKey = "rego_" + ast.HTTPSend.Name
```

Comment:
Would it make sense to define a shared builtin key prefix or something? Could maybe have a helper on the BuiltinContext or something.

🤔  That being said maybe we do it when we have >1, for a single metric it doesn't really matter too much. I just want to make sure we keep them uniform since they are (sort of) user facing names.

Reply:
Changed to `rego_builtin_http.send`.